website/search/elastic_search.py

```diff
@@ -336,11 +337,11 @@ def serialize_node(node, category):
         'id': node._id,
         'contributors': [
             {
-                'fullname': x.fullname,
-                'url': x.profile_url if x.is_active else None
+                'fullname': x['fullname'],
+                'url': '/{}/'.format(x['guids___id']) if x['is_active'] else None
             }
-            for x in node.visible_contributors
-            if x is not None
+            for x in node._contributors.filter(contributor__visible=True)
```

Comment:
Will these return contributors in the right order? You might need `.order_by('contributor___order')`.